docs/reference/method/MongoDBCollection-explain.txt

```diff
@@ -29,7 +29,7 @@ Definition
 Parameters
 ----------
 
-``$explainable`` : :phpclass:`MongoDB\\Operation\\Explainable`
+``$explainable`` : :phpclass:`MongoDB\Operation\\Explainable`
```

Comment:
@norareidy I think `\\Explainable` was missed here. Just caught it while merging v1.16 to v1.17 and grepping for "\\" in the docs folder.

Reply:
https://github.com/mongodb/mongo-php-library/pull/1244